vm/systemSmartContracts/delegation.go

```diff
@@ -561,6 +561,16 @@ func (d *delegation) modifyTotalDelegationCap(args *vmcommon.ContractCallInput)
 	return vmcommon.Ok
 }
 
+func (d *delegation) checkBLSKeysIfExistsInStakingSC(blsKeys [][]byte) bool {
```

Comment:
checkIfAtLeastOneBLSKeyExistsInStakingSC

Reply:
would leave as it is